DEPLOYMENT.md

```diff
@@ -0,0 +1,51 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wildcard SSL certificate be installed on the OpenShift router. However in situations where this is not desirable, RHMAP Core can be configured to use non-wildcard routing.
```

Comment:
` RHMAP Core can be configured to use non-wildcard routing`
How about:
`RHMAP can be exposed using single URL`?